test/e2e/serial/tests/configuration.go

```diff
@@ -170,12 +158,26 @@ var _ = Describe("[serial][disruptive] numaresources configuration management",
 					Timeout(configuration.MachineConfigPoolUpdateTimeout).
 					ForMachineConfigPoolDeleted(context.TODO(), mcp)
 				Expect(err).ToNot(HaveOccurred())
+
 			}()
+			//so far 0 machine count for mcp-test -> no nodes -> no updates status
+			newMcps := []*machineconfigv1.MachineConfigPool{mcp}
 
-			// save the initial nrop mcp to use it later while waiting for mcp to get updated
-			initialMcps, err := nropmcp.GetListByNodeGroupsV1(context.TODO(), e2eclient.Client, nroOperObj.Spec.NodeGroups)
+			By(fmt.Sprintf("Label node %q with %q", targetedNode.Name, nodes.GetLabelRoleMCPTest()))
+			unlabelFunc, err := labelNode(fxt.Client, nodes.GetLabelRoleMCPTest(), targetedNode.Name)
 			Expect(err).ToNot(HaveOccurred())
 
+			defer func() {
+				By(fmt.Sprintf("CLEANUP: restore initial labels of node %q with %q", targetedNode.Name, nodes.GetLabelRoleWorker()))
+				err = unlabelFunc()
+				Expect(err).ToNot(HaveOccurred())
+
+				//this will trigger node reboot as the NROP settings will be reapplied to the unlabelled node
+				waitForMcpUpdate(fxt.Client, context.TODO(), initialMcps)
```

Comment:
considering what we fixed like 1-2 commits before, do we need to check the error here?
`waitForMcpuUpdate` would need the changes to not swallow the errors

Reply:
this is done inside https://github.com/openshift-kni/numaresources-operator/pull/968/commits/ff19d4bae8dbe48ef2f4b70347a00c0193a0d1b7#diff-ff13df3dd632af928cfdb92fae23b8e0f6de4cfe44a77c0d3a5d1a5a2eacbf5dR786 `waitForMcpUpdate()` is basically waiting for complete updates (updating -> updated) for the **same** mcp group